string eventName
string date
bool isWeekly
---
string resultCode
